lib/rubocop/cli/command/auto_generate_config.rb

```diff
@@ -16,6 +16,7 @@ class AutoGenerateConfig < Base
 
         PHASE_1_OVERRIDDEN = '(skipped because the default Layout/LineLength:Max is overridden)'
         PHASE_1_DISABLED = '(skipped because Layout/LineLength is disabled)'
+        PHASE_1_SKIPPED = '(skipped because the COPs from the --only flag are parsed only)'
```

Comment:
I'm struggling to understand this sentence, so it should probably be reworded - e.g. what is "the cops from the --only flag are parsed only"?

Reply:
in case the `Layout/LineLength` COP is enabled, and we use the `--only` flag with auto-generate, it will only generate the config for the given COPs passed with the `--only` flag

Reply:
@istvanfazakas But I still don't get what is being "parsed" here. Is this some typo and you meant to write "passed"?

Reply:
sorry, yes, it seems to be a typo. I will fix it

Reply:
@bbatsov I just updated the sentence. Please take a look if it makes more sense now 🙏